src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -723,8 +730,20 @@ export const selectIsAnyTradeQuoteLoading = createSelector(
 )
 
 export const selectIsAnyTradeQuoteLoaded = createSelector(
-  selectIsSwapperQuoteAvailable,
   selectHasUserEnteredAmount,
-  (isSwapperQuoteAvailable, hasUserEnteredAmount) =>
-    !hasUserEnteredAmount || Object.values(isSwapperQuoteAvailable).some(identity),
+  selectIsAnyTradeQuoteLoading,
+  selectIsAnySwapperQuoteAvailable,
+  (hasUserEnteredAmount, isAnyTradeQuoteLoading, isAnySwapperQuoteAvailable) => {
+    if (!hasUserEnteredAmount) {
+      return true
+    }
```

Comment:
q: This is carried over from the previous implementation but it does look off. Can we actually have any trade quote loaded if user hasn't entered an amount?

Reply:
its more for paranoia than anything, which I'll comment 🙏